src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/request/StrikeOffObjectionsRequestEntity.java

```diff
@@ -0,0 +1,129 @@
+package uk.gov.companieshouse.api.strikeoffobjections.model.request;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
+
+import java.time.LocalDateTime;
+
+@Document(collection = "strike_off_objections_requests")
+public class StrikeOffObjectionsRequestEntity {
+
+
+    public static class Builder {
+        private String requestId;
+        private String username;
+        private String eMail;
+        private String requestInformation;
+        private LocalDateTime createdOn;
+        private RequestStatus status;
+
+        public Builder withRequestId(String requestId) {
+            this.requestId = requestId;
+            return this;
+        }
+
+        public Builder withUsername(String username) {
+            this.username = username;
+            return this;
+        }
+
+        public Builder withEMail(String eMail) {
+            this.eMail = eMail;
+            return this;
+        }
+
+        public Builder withRequestInformation(String requestInformation) {
+            this.requestInformation = requestInformation;
+            return this;
+        }
+
+        public Builder withCreatedOn(LocalDateTime createdOn) {
```

Comment:
Could this field not be automatically filled in with LocalDateTime.now() when the build() method is called? or is there a requirement to set the createdOn field for anything other than now()?

Reply:
We should delegate the responsibility for this to the service layer.